### PR TITLE
qa_crowbarsetup image code cleanup

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

**Comment by jdsn** [comment]
:+1:

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3577,10 +3587,7 @@ function oncontroller_run_tempest()
     fi
     local imageid=$(glance_image_get_id $image_name)
     crudini --set /etc/tempest/tempest.conf orchestration image_ref $imageid
-    # test if is cnftools image prepared for tempest
-    wait_for 300 5 \
-        'openstack image show $imageid | grep active &>/dev/null' \
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: here you're showing on $imageid

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -3577,10 +3587,7 @@ function oncontroller_run_tempest()
...
-        "prepare cnftools image"
+    wait_image_active "$image_name" tempest
```
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: and now you're showing $image_name, which might not necessarily work.. can we add imageid as a 1st parameter and use the name as a 2nd? I am not sure the "purpose" parameter is really needed.

Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: The change is intentional. We are getting the imageid value from a image show $image_name so it does not make any difference. And if the name was already found before, we do not upload again, so we do not get duplicate names.

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: hmm, okay. I am still not convinced, but am fine with it.

**Pull request merged**